tests/daemon/test-daemon.c

```diff
@@ -990,7 +957,7 @@ setup (Fixture      *fixture,
                           fixture->mock_network_send_provider,
                           fixture->mock_permissions_provider,
                           fixture->mock_persistent_cache,
-                          20); // Buffer length
+                          100000); // Max bytes buffered
```

Comment:
Maybe change this comment to the other style you use above while you're touching this?